spec/spec

```diff
@@ -742,15 +732,19 @@ beginseg
     include "$(BUILD_DIR)/src/libc64/malloc.o"
     include "$(BUILD_DIR)/src/libc64/qrand.o"
     include "$(BUILD_DIR)/src/libc64/__osMalloc_n64.o"
+#if !DEBUG_FEATURES
     include "$(BUILD_DIR)/src/libc64/sprintf.o"
+#endif
```

Comment:
I'm surprised this extra checks doesn't affect matching.

Reply:
It turns out some files were being passed to the linker twice and it was taking the first mention and ignoring the second. This broke partial linking though since two segments can end up with a copy of the files that would then cause final link to complain about multiple definitions.